ui/yarn.lock

```diff
@@ -1092,14 +1092,6 @@
   version "1.0.6"
   resolved "https://registry.yarnpkg.com/@types/geojson/-/geojson-1.0.6.tgz#3e02972728c69248c2af08d60a48cbb8680fffdf"
 
-"@types/glob@^7.1.1":
```

Comment:
A lot of changes here but it's about half of those in #5083 so I'd prefer this PR if it works